classifier/src/mentorpal/classifiers/arch/lstm_v1/__init__.py

```diff
@@ -62,7 +63,14 @@ def get_arch():
         global ARCH
         return ARCH
 
-    def get_answer(self, question):
+    def get_answer(self, question, canned_question_match_disabled=False):
+        if not canned_question_match_disabled:
+            sanitized_question = sanitize_string(question)
+            if sanitized_question in self.mentor.question_ids.keys():
```

Comment:
Is `mentor.question_ids` a `dict`? Does plain `if sanitized_question in self.mentor.question_ids:` need `.keys()` to work?